agent/handlers/v1_handlers_test.go

```diff
@@ -102,6 +102,13 @@ func TestServeHttp(t *testing.T) {
 	if tasks[0].Arn != "task1" {
 		t.Error("Incorrect task arn in response: ", tasks[0].Arn)
 	}
+	if tasks[0].KnownStatus != "STOPPED" {
+		t.Error("Incorrect known status in response: ", tasks[0].KnownStatus)
+	}
+	// Since the KnownStatus (STOPPED) > DesiredStatus (RUNNING), DesiredStatus should be empty
```

Comment:
Mind adding a second test that verifies the opposite, that it's present when it should be?